globjects-config.cmake

```diff
@@ -11,6 +11,10 @@ include(CMakeFindDependencyMacro)
 find_dependency(glm)
 find_dependency(glbinding)
 
+if(OPTION_USE_EIGEN)
+  find_dependency(eigen3 3.3 REQUIRED )
```

Comment:
Is the package written in lowercase or in uppercase (as used in the `find_package` call)?

Reply:
Right, it's uppercase, I will correct that tomorrow along with a check on double precision, which may be easy to add.
Array is another piece of cake because of alignment issue (and since I'm not an expert on that point, I will ask a co worker).

Reply:
Resolved.

Reply:
I just stumbled upon the `find_package` call in the configure script again.
Here, we depend on the cmake option `OPTION_USE_EIGEN` that is available during cmake configuration of the globjects project.
However, this file is evaluated when a downstream project, e.g., your project, is configured using cmake. The value of the option then does not depend on the value of the option when configuring and building globjects.
It seems an actual solution is code generation in this file, e.g., through use of the `configure_file` feature of cmake. However, as I'll want to integrate this PR soon, I'll just convert this call to a quiet dependency search.

This should be feasible as the decision whether to use eigen or not is compiled into the globjects library. From there, we four cases to consider:
1. globjects is build *without* eigen and a downstream project *does not* find eigen: no problem here, the behavior is just as before
2. globjects is build *without* eigen and a downstream project *does* find eigen: no problem, as eigen is not linked with globjects
3. globjects is build *with* eigen and a doenstream project *does not* find eigen: this issues a cmake error that the target `Eigen3::Eigen` is undefined
4. globjects is build *with* eigen and a doenstream project *does* find eigen: no problem here, this is the intended case if eigen should get used